packages/design-system/package.json

```diff
@@ -30,14 +30,14 @@
     "@fortawesome/free-solid-svg-icons": "5.x",
     "@fortawesome/vue-fontawesome": "2.x",
     "core-js": "3.x",
-    "element-ui": "2.13.x"
+    "element-ui": "2.15.x"
```

Comment:
you will need to update it in editor as well because it's a peer dependency https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/package.json#L65

Reply:
Done :)